pkg/plugins/runtime/k8s/webhooks/externalservice_validator.go

```diff
@@ -34,11 +34,11 @@ func (h *ExternalServiceValidator) InjectDecoder(d *admission.Decoder) error {
 
 func (h *ExternalServiceValidator) Handle(ctx context.Context, req admission.Request) admission.Response {
 	switch req.Operation {
-	case v1beta1.Delete:
+	case v1.Delete:
 		return h.ValidateDelete(ctx, req)
-	case v1beta1.Create:
+	case v1.Create:
 		return h.ValidateCreate(ctx, req)
-	case v1beta1.Update:
+	case v1.Update:
```

Comment:
What is the Kubernetes version impact here? Does dropping `admission/v1beta1` change the Kubernetes version support matrix?

Reply:
Same comment for all the validators. Do we need to make a pass a consolidate duplicated code here?

Reply:
I think it will affect the k8s support matrix, but admission api was promoted to `v1` in k8s [`1.16`](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.16.md#whats-new-major-themes) which is 1.5 years old.

Also in 1.17 it was made mandatory to use `v1` instead of `v1beta1`: https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.17.md#api-machinery

Reply:
If we are not supporting v1beta1, let's remove this from HELM deployment YAMLs